.changeset/pretty-turtles-leave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sunodo/cli": minor
```

Comment:
As it does not change the behavior for the end user, and it's mostly a patch to a potential issue, maybe this should be a patch?